conda/mpi/build.sh

```diff
@@ -16,7 +16,7 @@ mkdir -p "${PREFIX}/etc/conda/activate.d" "${PREFIX}/etc/conda/deactivate.d"
 cat <<EOF > "${PREFIX}/etc/conda/activate.d/activate_${PKG_NAME}.sh"
 export CC=mpicc CXX=mpicxx FC=mpif90 F90=mpif90 F77=mpif77 C_INCLUDE_PATH=${PREFIX}/include \
  MOOSE_NO_CODESIGN=true MPIHOME=${PREFIX} CXXFLAGS="$ACTIVATION_CXXFLAGS" HDF5_DIR=${PREFIX} \
- FI_PROVIDER=tcp
+ FI_PROVIDER=tcp OMPI_MCA_mca_base_component_show_load_errors=0
```

Comment:
This only needs to be set for openmpi, right?

Reply:
yes. And the FI_PROVIDER option only needs to be set for mpich

Reply:
is it harmless either way? Or should we only set it for one variant or another?

Reply:
I'm not quite sure what would be a good way to detect mpich vs. openmpi within this script

Reply:
The way to do this, is how I mistakenly instructed how to do it in another comment on this PR above...

Reply:
I'm fine with ignoring this. It's harmless